core/templates/components/state-editor/state-editor-properties-services/state-editor.service.ts

```diff
@@ -201,15 +204,17 @@ export class StateEditorService {
     return cloneDeep(this.interaction);
   }
 
+  // Function will return null if interactionId does not exist.
   getAnswerChoices(
       interactionId: string,
-      customizationArgs: InteractionCustomizationArgs): AnswerChoice[] {
+      customizationArgs: InteractionCustomizationArgs): AnswerChoice[] | null {
```

Comment:
```suggestion
      customizationArgs: InteractionCustomizationArgs
  ): AnswerChoice[] | null {
```

Reply:
Also is invalid interaction ID expected here? If not then maybe throw an error instead.

Reply:
The function will still return null if InteractionId is not among any of these values ` 'MultipleChoiceInput', 'ItemSelectionInput', 'DragAndDropSortInput'` so throwing an error here and updated the comment